src/main/java/com/rummikub/game/Game.java

```diff
@@ -0,0 +1,84 @@
+package com.rummikub.game;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Game {
```

Comment:
nit: class-level javadoc comments are also quite useful, even if they are kind of self explanatory i guess.